### PR TITLE
Documentation for PR #797 - Opentelemetry Collector Data Flow Dashboard

### PR DESCRIPTION
**Pull request by devrimdemiroz**
Documentation for [PR #797 ](https://github.com/open-telemetry/opentelemetry-demo/pull/797#issuecomment-1478615243) on opentelemetry demo application repo.

### CODE REVIEW

File: content/en/docs/demo/collector-data-flow-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+- scrape_samples_scraped
+- by (job)
+
+## Dashboard
+
+![](screenshots/otelcol-data-flow-dashboard.png)
+
+You can access the dashboard by navigating to the Grafana UI, selecting the
+"OpenTelemetry Collector Data Flow" dashboard under brose icon on the left-hand
```
Comment (julianocosta89) [inline content/en/docs/demo/collector-data-flow-dashboard.md:76]:
```suggestion
"OpenTelemetry Collector Data Flow" dashboard under browse icon on the left-hand
```

File: content-modules/community
Comment (svrnm) [inline content-modules/community:1]: Please make sure that changes to the content-modules are not checked in

Comment (devrimdemiroz) [reply content-modules/community:1]: I will create from scratch, got it. Thanks.

**Comment by devrimdemiroz** [comment]
opened this one --> https://github.com/open-telemetry/opentelemetry.io/pull/2538

File: content/en/docs/demo/collector-data-flow-dashboard.md (hunk 1 above)
Comment (devrimdemiroz) [reply content/en/docs/demo/collector-data-flow-dashboard.md:76]: > 

corrected in https://github.com/open-telemetry/opentelemetry.io/pull/2538

**Pull request closed without merging**